src/pause-play/js/main.js

```diff
@@ -66,18 +79,21 @@ class PeerConnection {
       this.remoteView.srcObject = e.stream;
     };
 
-    var onCreateOfferSuccess = this.onCreateOfferSuccess.bind(this);
     this.localConnection
       .createOffer({offerToReceiveAudio: 1, offerToReceiveVideo: 1})
-      .then(onCreateOfferSuccess, logError);
+      .then((desc) => {
+        this.onCreateOfferSuccess(desc);
+      }, logError);
   }
 
   onCreateOfferSuccess(desc) {
     this.localConnection.setLocalDescription(desc);
     this.remoteConnection.setRemoteDescription(desc);
 
-    var onCreateAnswerSuccess = this.onCreateAnswerSuccess.bind(this);
-    this.remoteConnection.createAnswer().then(onCreateAnswerSuccess, logError);
+    this.remoteConnection.createAnswer().then(
+      () => {
```

Comment:
shouldn't you add desc as an argument? Also format it the same as on line https://github.com/webrtc/test-pages/pull/7/files#diff-2edb0afc11050f78b4acd671ec919f5eR84